### PR TITLE
Add support for Layout/Grid titles in bokeh

### PR DESCRIPTION
**Pull request by philippjfr**
As I had suggested in #989 it is possible to add a Div as a title to a composite plot. This finally adds consistency in titles between matplotlib and bokeh backends for Layout and Grid plots.

Still needs tests.

### Layout

screen shot 2016-12-11 at 10 24 55 pm

### Grid

screen shot 2016-12-11 at 10 27 19 pm

### CODE REVIEW

**Comment by jlstevens** [comment]
That looks good! Shame bokeh doesn't let you do this more directly.

Make sure to test this in Chrome/Firefox/Safari and if you find it is working well, I am happy to merge.

**Comment by philippjfr** [comment]
Ready to merge when tests pass.

File: holoviews/plotting/bokeh/plot.py [hunk 1]
```diff
@@ -153,7 +154,43 @@ def sync_sources(self):
 
 
 
-class GridPlot(BokehPlot, GenericCompositePlot):
+class CompositePlot(BokehPlot):
+
```
Comment (jlstevens) [inline holoviews/plotting/bokeh/plot.py]: Would be good to have a docstring explaining what this base class is for. Right now, everything ``CompositePlot`` does seems to be only needed by ``LayoutPlot``. Any ideas what ``CompositePlot`` could be used for that isn't ``LayoutPlot``?

Comment (philippjfr) [inline holoviews/plotting/bokeh/plot.py]: GridPlot, just below.

**Comment by jlstevens** [comment]
Looks good, thanks!

**Pull request merged**